api-integration/smart-deposits.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: 'Smart Deposit Addresses'
+description: 'Smart Deposit Addresses allow you to generate unique addresses that automatically bridge assets to a specified destination upon receiving a transfer.'
+---
+
+import DepositAddressDisclaimer from '/snippets/deposit-address-disclaimer.mdx'
+
+<DepositAddressDisclaimer />
+
+## Introduction
+
+To bridge funds through Rhino you would normally need to generate a quote, commit it and then interact with the bridge contract on the origin chain to initiate the bridge. However, this is not always an option for all use cases where a smart contract interaction is not possible or very inconvenient.  
+To also cater to those use casas, Rhino supporting bridging through manual transfers.
```

Comment:
` Rhino supporting bridging through manual transfers.`, missing an is or something